e2e-tests/steps/apply.ts

```diff
@@ -103,4 +103,12 @@ export const addNote = async (page: Page) => {
   await page.getByTestId('submit-button').click()
   await expect(page.locator('h2').first()).toContainText('Success')
   await expect(page.locator('.moj-timeline__description').first()).toContainText(note)
+  await expect(page.locator('.moj-timeline__title').first()).toContainText('Note')
+}
+
+export const viewSubmittedApplicationOverview = async (applicationId: string, page: Page, name: string) => {
```

Comment:
ooh maybe small thing but could this be renamed to be more specific? `viewApplicationFromPrisonDashboard` or something?